blinkit-app/src/DATA/ITEMS.js

```diff
@@ -0,0 +1,165 @@
+const items=[
+    {
+        "itemId":0,
```

Comment:
Give appropriate ids, as id is a unique identifier, here 0 can overlap with other items in the DOM